### PR TITLE
Changed the getKeyStroke functionality, modified it's test cases, add…

### PR DESCRIPTION
**Pull request by harshlodhi82**
…ed settings.ts, added executable path on browser launch, changed the start.ts for the changed

### CODE REVIEW

File: chat/lib/Chat/getKeystrokeFromMessage/getKeystrokeFromMessage.test.ts [hunk 1]
```diff
@@ -6,46 +6,47 @@ test('Chat.getKeystrokeFromMessage', () => {
...
+      '2': ['2'],
+      a: ['A'],
+      'jump': ['shift', 'j']
     }
   })
 
   let res
 
   res = chat.getKeystrokeFromMessage({username, message: 'up'})
-  expect(res).toEqual({key: 'u', duration})
```
Comment (dave-dm) [inline chat/lib/Chat/getKeystrokeFromMessage/getKeystrokeFromMessage.test.ts:21]: `key` should be renamed to `keys`

File: chat/lib/Chat/getKeystrokeFromMessage/index.ts [hunk 2]
```diff
@@ -27,13 +27,12 @@ const _isValidArgs = (username, message) => {
 }
 
 const _returnValue = (self, message: string, validKey: string, duration: number): Keystroke => {
```
Comment (dave-dm) [inline chat/lib/Chat/getKeystrokeFromMessage/index.ts:29]: _returnValue is not a good function name, it doesn't say what it does

File: chat/lib/Chat/sendKeystrokeToWindow/index.ts [hunk 3]
```diff
@@ -3,8 +3,8 @@ import keySender from 'node-key-sender'
 
 const sendKeystrokeToWindow = async function (keystroke: Keystroke) {
   try {
-    let keys = keystroke.key.split("")
-    let resData = await keySender.sendKeys(keys)
+    let keys = keystroke.key
```
Comment (dave-dm) [inline chat/lib/Chat/sendKeystrokeToWindow/index.ts:6]: you need to change the interface of `Keystroke` to {keys: string[], duration: number}. Don't just rename it in one place, it needs to be changed everywhere.

File: chat/lib/Chat/getKeystrokeFromMessage/index.ts (hunk 2 above)
Comment (harshlodhi82) [reply chat/lib/Chat/getKeystrokeFromMessage/index.ts:29]: Now we don't need the _returnValue function so i'm going to remove it,

**Pull request merged**